docs/resources/service_principal_secret.md

```diff
@@ -25,14 +25,20 @@ resource "databricks_service_principal_secret" "terraform_sp" {
 
 The following arguments are available:
 
-* `service_principal_id` - ID of the [databricks_service_principal](service_principal.md) (not application ID).
+* `service_principal_id` (Required, string) - SCIM ID of the [databricks_service_principal](service_principal.md) (not application ID).
+* `lifetime` (Optional, string) - The lifetime of the secret in seconds formatted as `NNNNs`. If this parameter is not provided, the secret will have a default lifetime of 730 days (`63072000s`).  Expiration of secret will lead to generation of new secret.
 
 ## Attribute Reference
 
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - ID of the secret
-* `secret` - Generated secret for the service principal
+* `secret` - Generated secret for the service principal.
+* `create_time` - UTC time when the secret was created.
+* `expire_time` - UTC time when the secret will expire. If the field is not present, the secret does not expire.
+* `secret_hash` - Secret Hash.
+* `status`  - Status of the secret (i.e., `ACTIVE`).
```

Comment:
Might be good to list all the enum values here the `status` can take

Reply:
Right now it's only `ACTIVE`